eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1807,8 +1810,11 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
   def handleOfflineFeerate(c: CurrentFeerates, d: HasCommitments) = {
     val networkFeeratePerKw = c.feeratesPerKw.feePerBlock(target = nodeParams.onChainFeeConf.feeTargets.commitmentBlockTarget)
     val currentFeeratePerKw = d.commitments.localCommit.spec.feeratePerKw
-    // if the fees are too high we risk to not be able to confirm our current commitment
-    if (networkFeeratePerKw > currentFeeratePerKw && Helpers.isFeeDiffTooHigh(currentFeeratePerKw, networkFeeratePerKw, nodeParams.onChainFeeConf.maxFeerateMismatch)) {
+    // if the network fees are too high we risk to not be able to confirm our current commitment
+    val shouldClose = networkFeeratePerKw > currentFeeratePerKw &&
+      Helpers.isFeeDiffTooHigh(networkFeeratePerKw, currentFeeratePerKw, nodeParams.onChainFeeConf.maxFeerateMismatch) &&
+      d.commitments.hasPendingOrProposedHtlcs // we close only if we have HTLCs potentially at risk
+    if (shouldClose) {
       if (nodeParams.onChainFeeConf.closeOnOfflineMismatch) {
         log.warning(s"closing OFFLINE ${d.channelId} due to fee mismatch: currentFeeratePerKw=$currentFeeratePerKw networkFeeratePerKw=$networkFeeratePerKw")
```

Comment:
(same for the comment a few lines below)

```suggestion
        log.warning(s"closing OFFLINE channel due to fee mismatch: currentFeeratePerKw=$currentFeeratePerKw networkFeeratePerKw=$networkFeeratePerKw")
```

Reply:
Fixed in https://github.com/ACINQ/eclair/pull/1473/commits/9037da2978266fb0de40bf13c272d0024dfedcea